test/common/vmmanage.py

```diff
@@ -46,6 +46,10 @@ def run_install_script(machine, do_build, do_install, skips, arg, args):
     if install and args["containers"]:
         machine.execute("/var/lib/testvm/containers.install")
 
+    # TMP HACK FOR SELINUX
```

Comment:
Obviously with a HACK linking to the appropriate bugzilla bug containing the request to change the selinux-policy?